src/katgpucbf/xbgpu/kernels/tensor_core_correlation_kernel.cu

```diff
@@ -41,7 +41,7 @@
  *
  * SARAO's modification is licenced as follows:
  *******************************************************************************
- * Copyright (c) 2020-2021, National Research Foundation (SARAO)
```

Comment:
I'd also suggest adding a bullet point about the saturation detection in the above list.

Reply:
It's not part of the modified TCC kernel - it's a separate kernel that is tacked onto the end of the file.

Reply:
Ah yes of course. Disregard.